Generator/ActionNamesInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Pgs\RestfonyBundle\Generator;
+
+interface ActionNamesInterface
```

Comment:
IMHO would be better to use pure `class` or `final class` in this case,
e.g `ActionNames` without suffix